### ISSUE TITLE
[DOXIASITETOOLS-8] The order of <title> tag and <meta> tag

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-doxia-sitetools#312]
**[Jun Futagawa](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=jfut)** opened **[DOXIASITETOOLS-8](https://issues.apache.org/jira/browse/DOXIASITETOOLS-8?redirect=false)** and commented

When there are characters other than ASCII in the \<title> tag before the \<meta> tag for charset appears, The automatic character-code recognition of IE7 fails occasionally, and the empty page is displayed.

For this reason, I wish default-site.vm becomes the following.

```
...
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=${outputEncoding}" />
    <title>$title</title>
...
```

---

**Affects:**  1.0-alpha-10

**Backported to:** [1.0](https://github.com/apache/maven-doxia-sitetools/milestone/5?closed=1)

**Comment by jira-importer** [comment]
**[Lukas Theussl](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=ltheussl)** commented

Fixed in r732684.

**Comment by jira-importer** [comment]
**[Dennis Lundberg](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dennisl@apache.org)** commented

Fixed for 1.0-branch in r737617.
